src/components/Organization/pages/Users.jsx

```diff
@@ -15,6 +21,28 @@ const useStyles = makeStyles(theme => ({
 
 function Users() {
   const classes = useStyles();
+  const [modalOpenAdmin, setModalOpenAdmin] = useState(false);
```

Comment:
You are not utilising any of these states in this component expect for sending them as props right, so i suggest you redefine all the states related modal in OrgUsers component itself.
This will:
1. Improve locality of the states and is easy to track their functionality
2. Avoid un-necessary re-renders in this component due to the state change of states related to some other component